netbox/ipam/forms/bulk_edit.py

```diff
@@ -203,25 +203,6 @@ class RoleBulkEditForm(NetBoxModelBulkEditForm):
 
 
 class PrefixBulkEditForm(NetBoxModelBulkEditForm):
```

Comment:
should we add scope here?

Reply:
I think we run into a snag using HTMXSelector on a bulk edit form: The bulk edit forms must be POSTed to capture the selected object PKs, whereas HTMXSelector uses a GET to re-render the form. We'll need to extend HTMXSelector to allow POSTing instead.

Reply:
Going to work on #17789 first to get a pattern established in `develop`, then port it over here.